buildbot_nix/github_projects.py

```diff
@@ -585,10 +585,15 @@ def create_reporter(self) -> ReporterBase:
         return self.auth_backend.create_reporter()
 
     def create_change_hook(self) -> dict[str, Any]:
+        def get_github_token(props: Properties) -> str:
+            return self.auth_backend.get_repo_token(
+                props.getProperty("full_name")
+            ).get()
+
         return {
             "secret": self.webhook_secret,
             "strict": True,
-            "token": self.auth_backend.get_general_token().get(),
```

Comment:
Do we still need this method afterwards?

Reply:
Ah good catch we don't, is there some "dead code" thing for python? Would require us to keep explicit exports, which is doable